### ISSUE TITLE
test strings with a number are not recognized

### ISSUE THREAD

**Issue by thisconnect** [scottcorgan/tap-out#14]
using something like `tape test.js`
```
TAP version 13
# commit one time
ok 1 all good?
# commit times2
ok 2 still good?
# commit 3 times
ok 3 WHOOT!
# commit *4 times
ok 4 good
```
with tap-spec `tape test.js | tap-spec` will output something like
```
  commit one time

    ✔ all good?

  commit times2

    ✔ still good?
    ✔ WHOOT!

  commit *4 times

    ✔ good
```

It seems as the result regex is matching, and result test is before type test check.
https://regex101.com/r/iF6vM2/2

**Comment by scottcorgan** [comment]
I have been working on a collection of packages under the org [tap-format](https://github.com/tap-format) as sequels to my few [tap-[formatter]](https://github.com/search?utf8=%E2%9C%93&q=repo%3Ascottcorgan%2Ftap-spec+repo%3Ascottcorgan%2Ftap-dot+repo%3Ascottcorgan%2Ftap-out&type=Repositories&ref=searchresults) modules.

https://github.com/tap-format/parser will be the replacement for this. It's basically done, just needs documentation.

**Comment by thisconnect** [comment]
Will the neq parser be used in tap-spec?

**Comment by scottcorgan** [comment]
Yes it will. Except that eventually, tap-spec will go into maintenance mode in favor of it's predecessor: https://github.com/tap-format/spec. This module will do the same exact thing, except cleaner output and more stable.
